styles/_fonts.less

```diff
@@ -82,11 +82,16 @@
   font-size: 40px;
 }
 
-.fontL, .fontM, .fontStaticM {
+.fontL, .fontM {
   font-size: 30px;
   line-height: 1.2em;
 }
 
+.fontStaticM {
```

Comment:
So the idea of `static` font size was that it doesn't actually change and persists through viewport changes. This change sort of defeats that purpose.